open_spiel/python/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -59,8 +59,7 @@
 class TrajectoryState(object):
   """A particular point along a trajectory."""
 
-  def __init__(self, observation, current_player, legals_mask, action, policy,
-               value):
+  def __init__(self, observation, current_player, legals_mask, action, policy, value):
```

Comment:
We follow the google style guide: https://google.github.io/styleguide/pyguide.html , which has an 80 char line limit.